bin/fastlane

```diff
@@ -4,6 +4,30 @@ if RUBY_VERSION < '2.0.0'
   abort("fastlane requires Ruby 2.0.0 or higher")
 end
 
+def self.windows?
+  # taken from: https://stackoverflow.com/a/171011/1945875
+  (/cygwin|mswin|mingw|bccwin|wince|emx/ =~ RUBY_PLATFORM) != nil
+end
+
+require 'bootsnap'
+require 'tmpdir'
+require 'fileutils'
+# env = ENV['RAILS_ENV'] || "development"
+require_relative "../fastlane/lib/fastlane/version.rb"
+cache_dir = "#{Dir.tmpdir}/fastlane/bootsnap-cache/" + Fastlane::VERSION
+unless File.directory?(cache_dir)
+  FileUtils.mkdir_p(cache_dir)
+end
+Bootsnap.setup(
+  cache_dir:            cache_dir, # Path to your cache
+  development_mode:     false, # env == 'development', # Current working environment, e.g. RACK_ENV, RAILS_ENV, etc
```

Comment:
Can probably remove all of the comments on this line? `RACK_ENV` and `RAILS_ENV` are not relevant at all

Reply:
Yes, see previous comment.

I left the `development_mode` stuff in, as fastlane doesn't seem to have something like this - and this would be one of the cases where it would probably be useful. (Although I didn't have any negative effects myself while developing stuff with `bootsnap` enabled)

Reply:
I think its okay to leave the development mode stuff in but remove the ` e.g. RACK_ENV, RAILS_ENV, etc` as these are specific to web frameworks which fastlane is not 😉